### DISCUSSION TITLE
GitHub Mobile - Android - Check Status showing incorrect values

### DISCUSSION THREAD

**Discussion by setchy** [community/community#123211] [Mobile]
I've noticed for a while that the check status / GitHub actions status results are showing incorrectly.

It appears to show the first check status results and not the latest results status.

**Comment by msi-end** [comment]
* **Refresh the Page:** Sometimes, the app might not automatically refresh the status results. Try manually refreshing the page to see if the latest status updates are reflected.
    
* **Check Internet Connection:** Ensure that your device has a stable internet connection. If the connection is intermittent or slow, it might prevent the app from fetching the latest status updates.
    
* **Clear App Cache:** Clearing the cache for the GitHub Mobile app can sometimes resolve display issues. You can do this by going to your device's settings, finding the GitHub app in the list of installed apps, and selecting the option to clear the cache.
    
* **Update the App:** Make sure that you're using the latest version of the GitHub Mobile app. Developers often release updates to fix bugs and improve performance, so updating the app might resolve the issue.
    
* **Report the Issue:** If none of the above steps resolve the problem, consider reporting the issue to GitHub support or via their official GitHub repository. Providing details about your device, app version, and steps to reproduce the issue can help the developers investigate and fix the problem.
